userguide/one_shot_object_detection/README.md

```diff
@@ -96,7 +88,7 @@ To learn more about deploying One-Shot to CoreML details, refer to the [Object D
 
 **Background images**
```

Comment:
I would add a section called Advanced usage and then show this + some of the other things in the user guide.

Reply:
This section 'Background images' is under the section 'Advanced Usage'. Is there anything else you feel we should add to Advanced Usage?